.woodpecker.star

```diff
@@ -947,7 +955,7 @@ def localApiTestPipeline(ctx):
 
 def localApiTests(ctx, name, suites, storage = "decomposed", extra_environment = {}, with_remote_php = False):
     test_dir = "%s/tests/acceptance" % dirs["base"]
-    expected_failures_file = "%s/expected-failures-localAPI-on-%s-storage.md" % (test_dir, storage)
+    expected_failures_file = "%s/expected-failures-localAPI-on-decomposed-storage.md" % (test_dir)
```

Comment:
I know I am too late to review, but I think we will be in trouble here if we hard-code the file-name.
If that is the correct name for all storages, maybe better to remove the storage from the file name

Reply:
np, I'll do it in the next PR